### PR TITLE
Prepackage focalboard v0.15.0

### PR DESCRIPTION
**Pull request by sbishel**
#### Summary
Update prepackaged boards version to 0.15.0

#### Ticket Link
Fixes https://github.com/mattermost/focalboard/issues/2350
  
#### Release Note
```release-note
Update prepackaged boards version to 0.15.0
```

### CODE REVIEW

**Comment by sbishel** [comment]
Waiting for final release before merge.

**Review by chenilim — APPROVED** [review]

**Review by harshilsharma63 — APPROVED** [review]

**Pull request merged**
